packages/experiments/src/Foundation.ts

```diff
@@ -77,20 +118,21 @@ export function createComponent<TComponentProps, TViewProps, TStyleSet extends I
     ITheme,
     ISchemeNames,
     TStatics
-  >(options, providers);
+  >(component, providers);
 }
 
 function getCustomizations<TViewProps, TStyleSet>(
   displayName: string,
-  context: ICustomizerContext
+  context: ICustomizerContext,
+  fields?: string[]
 ): IStyleableComponentProps<TViewProps, TStyleSet, ITheme, ISchemeNames> {
-  // TODO: do we want field props? should fields be part of IComponentOptions and used here?
+  // TODO: do we want field props? should fields be part of IComponent and used here?
   // TODO: should we centrally define DefaultFields? (not exported from styling)
   const DefaultFields = ['theme', 'styles'];
-  return Customizations.getSettings(DefaultFields, displayName, context.customizations);
+  return Customizations.getSettings(fields || DefaultFields, displayName, context.customizations);
```

Comment:
Perhaps default assignment here?

Reply:
Thanks for the comment. I need to consider whether I finalize the fields arg as optional or required as part of this PR.